src/schemas.d.ts

```diff
@@ -1647,6 +1647,161 @@ export interface Address {
   country?: string;
 }
 
+/**
+ * Schema defining the data produced by the Front-Office when an LPA Questionnaire is submitted
+ */
+export interface LPAQuestionnaireCommand {
+  casedata: {
+    /**
+     * External case identifier
+     */
+    caseReference: string;
+    /**
+     * The date the LPA provided a response to the case
+     */
+    lpaQuestionnaireSubmittedDate: string | null;
+    /**
+     * A statement provided by the LPA
+     */
+    lpaStatement: string | null;
```

Comment:
not sure we ask for an lpa statement, unless I'm we've called this something else